Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -0,0 +1,227 @@
+                    Button("Clear", systemImage: "trash.fill") {
+                        model.removeAllGraphics()
+                    }
+                }
+            }
+            .errorAlert(presentingError: $error)
+    }
+}
+
+private extension ShowServiceAreaView {
+    // MARK: - GraphicType
+    
+    enum GraphicType: Equatable, CaseIterable {
+        case facilities, barriers
+        
+        /// The string representation of this graphic type.
+        var label: String {
+            switch self {
+            case .barriers: "Barriers"
+            case .facilities: "Facilities"
+            }
+        }
+    }
+}
+
+private extension ShowServiceAreaView {
+    // MARK: - Model
+    
+    class Model: ObservableObject {
+        /// A map with terrain style centered over San Diego.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISTerrain)
+            map.initialViewpoint = Viewpoint(
+                center: Point(
+                    x: -13041154,
+                    y: 3858170,
+                    spatialReference: .webMercator
+                ),
+                scale: 60_000
+            )
+            return map
+        }()
+        
+        let facilitiesGraphicsOverlay: GraphicsOverlay = {
+            let facilitiesGraphicsOverlay = GraphicsOverlay()
+            let facilitySymbol = PictureMarkerSymbol(image: UIImage(named: "PinBlueStar")!)
+            // Offsets symbol in Y to align image properly.
+            facilitySymbol.offsetY = 21
+            // Assigns renderer on facilities graphics overlay using the picture marker symbol.
+            facilitiesGraphicsOverlay.renderer = SimpleRenderer(symbol: facilitySymbol)
+            return facilitiesGraphicsOverlay
+        }()
+        
+        let barriersGraphicsOverlay: GraphicsOverlay = {
+            let barriersGraphicsOverlay = GraphicsOverlay()
+            let barrierSymbol = SimpleFillSymbol(style: .diagonalCross, color: .red, outline: nil)
+            // Sets symbol on barrier graphics overlay using renderer.
+            barriersGraphicsOverlay.renderer = SimpleRenderer(symbol: barrierSymbol)
+            return barriersGraphicsOverlay
+        }()
+        
+        let serviceAreaGraphicsOverlay = GraphicsOverlay()
+        
+        /// The graphics overlays used by this model.
+        var graphicsOverlays: [GraphicsOverlay] {
+            return [facilitiesGraphicsOverlay, barriersGraphicsOverlay, serviceAreaGraphicsOverlay]
+        }
+        
+        private let serviceAreaTask = ServiceAreaTask(url: .serviceArea)
+        
+        private var serviceAreaParameters: ServiceAreaParameters!
+        
+        /// On user tapping on screen it add a facility graphic to the facilities overlay on the map at that point.
+        /// - Parameter point: The coordinates for the graphic.
+        func addFacilityGraphic(at point: Point) {
+            let graphic = Graphic(geometry: point, symbol: nil)
```

Comment:
`symbol` has a default value of `nil`:

```suggestion
            let graphic = Graphic(geometry: point)
```